exercises/07-automate-deployment.md

```diff
@@ -38,7 +38,6 @@ Those files are necessary to run the pipeline with Travis CI:
 
 ## Adjust pipeline configuration
 Go to the [pipeline_config.yml](pipeline_config.yml) file and adjust it to your project. Set the org and space for the `productionDeployment` as you have done before when logging in to Cloud Foundry (if you don't remember run `cf target` on the command line to get a summary).
-Also, replace your participantId for the `appName`.
```

Comment:
Why did you remove this sentence? It could be slightly improved, but I think this is very relevant, so nobody forgets.

Suggestion: Also, replace your participantId in the `appName`.

Reply:
One more thing: I realized that most of the links on this page (and potentially others) are incorrect: [pipeline_config.yml](pipeline_config.yml) should be [pipeline_config.yml](../pipeline_config.yml).
I created a ticket for that, but if you can, maybe you can make this a part of this PR already.